### ISSUE TITLE
Since KiTTy 0.76, something is wrong with window focus...

### ISSUE THREAD

**Issue by meyergru** [mRemoteNG/mRemoteNG#2094]
## Expected Behavior
The keyboard focus should become active in the KiTTy SSH window that I have clicked into.

## Current Behavior
Sometimes, after I upgraded to KiTTy 0.76 and mRemoteNG nightly, the focus seems to get lost.

For example, when I click on a tab (and on a tab only, not into the corresponding window, the focus does not change to the window. The tab itself becomes highlighted, but the cursor in the window does noch change to solid.

Sometimes, when a process takes longer in the SSH window and I change focus to another application, when I come back, the process seems stuck and only after I click on the scroll bar, the output becomes visible.

## Possible Solution
I am at a loss, even to if this is attributable to mRemoteNG, newer PuTTY (which I do not use) or updated KiTTy.

## Steps to Reproduce (for bugs)
See above, but this often occurs when the window is "full", i.e. you can scroll up already.

Alas, I can give no reproducible steps, the bug seems timing-related.

## Context
This is very annoying, since it often takes multiple actions to switch to another tab (or even to the currently open one when coming from another application).

## Your Environment
* Version used mRemoteNg 1.77.2.28860, KiTTy 0.76.0.2
* Operating System and version (Windows 10 Pro x64 21H1 19043.1415)

**Comment by farosch** [comment]
Closing, follow #1535 instead.
